service/qos/standard_service_level_distributed_data_accessor.cc

```diff
@@ -24,11 +24,11 @@ future<qos::service_levels_info> standard_service_level_distributed_data_accesso
     return _sys_dist_ks.get_service_level(service_level_name);
 }
 
-future<> standard_service_level_distributed_data_accessor::set_service_level(sstring service_level_name, qos::service_level_options slo, std::optional<service::group0_guard>) const {
+future<> standard_service_level_distributed_data_accessor::set_service_level(sstring service_level_name, qos::service_level_options slo, std::optional<service::group0_guard>, abort_source*) const {
```

Comment:
if abort_source is mandatory a reference would express it better

Reply:
it is not mandatory in the legacy service levels (standard_service_level_distributed_data_accessor)